cost-analyzer/templates/cost-analyzer-deployment-template.yaml

```diff
@@ -463,6 +463,7 @@ spec:
                   key: prometheus-server-endpoint
             - name: CLOUD_PROVIDER_API_KEY
               value: "AIzaSyDXQPG_MHUEy9neR7stolq6l0ujXmjJlvk" # The GCP Pricing API requires a key.
+                                                               # This GCP api key is expected to be here and is limited to accessing google's billing API. 
```

Comment:
Please see comments here: https://github.com/kubecost/cost-analyzer-helm-chart/pull/1726/files

@AjayTripathy

Reply:
@dwbrown2  That PR has been closed , please consider this PR

Reply:
@dwbrown2 Apologies, I overlooked the comment login to this repo was my personal email id.
I have made a one-liner statement could you review it and let me know if it looks good?